converters/toZigbee.js

```diff
@@ -1457,37 +1457,90 @@ const converters = {
             await entity.command('genOnOff', 'toggle', {}, {transactionSequenceNumber: 0});
         },
     },
+    livolo_cover_state: {
+        key: ['state'],
+        convertSet: async (entity, key, value, meta) => {
+            let payload;
+            const options = {frameType: 0, manufacturerCode: 0x1ad2, disableDefaultResponse: true,
+                disableResponse: true, reservedBits: 3, direction: 1, writeUndiv: true,
+                transactionSequenceNumber: 0xe9};
+            switch (value) {
+            case 'OPEN':
+                payload =
+                    {attrId: 0x0000, selector: null, elementData: [0x04, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00]};
+                break;
+            case 'CLOSE':
+                payload =
+                    {attrId: 0x0000, selector: null, elementData: [0x10, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00]};
+                break;
+            case 'STOP':
+                payload =
+                    {attrId: 0x0000, selector: null, elementData: [0x08, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00]};
+                break;
+            default:
+                throw new Error(`Value '${value}' is not a valid cover position (must be one of 'OPEN' or 'CLOSE')`);
```

Comment:
Should this also include 'STOP' as that seems to be a valid value as there is a case and payload for it above. (Although yes, STOP would not be a valid 'position' it is still a valid value)

Reply:
That's true